app.R

```diff
@@ -261,6 +268,28 @@ server <- function(input, output) {
                                                                                     levels = c("Not very likely", "Somewhat likely", "Likely", "Very likely", "Extremely likely")))
   })
   
+  # Course Engagement by modality 
+  course_raw <- reactiveFileReader(time_interval,
+                                   NULL,
+                                   "https://docs.google.com/spreadsheets/d/1-8vox2LzkVKzhmSFXCWjwt3jFtK-wHibRAq2fqbxEyo/edit?usp=sharing",
+                                   googlesheets4::read_sheet,
+                                   sheet = "engagement overall") 
+  
+  
+  course_processed <- reactive({
+    course_raw() %>% 
+      pivot_longer(cols = contains("count"), names_to = "modality", values_to = "number_of_learners") %>%
+      mutate(course_name = factor(course_name)) %>%
+      separate(modality, sep = "_", into = c("modality", "meh")) %>% 
+      mutate(modality = factor(modality, levels = c("website", "leanpub", "coursera")))%>%
```

Comment:
```suggestion
      mutate(modality = factor(modality, levels = c("website", "leanpub", "coursera")))%>%
```

Can we have Leanpub and Coursera capitalized? Not sure where we change this.